indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Integer.kt

```diff
@@ -35,7 +35,7 @@ fun Asn1Integer(number: ULong) =
  * It has a [sign] though, and supports [twosComplement] representation and converting [fromTwosComplement].
  * Hence, it directly interoperates with [Kotlin MP BigNum](https://github.com/ionspin/kotlin-multiplatform-bignum) and the JVM BigInteger.
  */
-@Serializable(with = Asn1IntegerSerializer::class)
+@Serializable(with = Asn1IntegerSerializer::class) //this here stays, as it might be useful in general
```

Comment:
Why just for `Asn1Integer`s?

Reply:
good point! should also cover the real type!

Reply:
done

Reply:
Because the numeric types might be generally usable if you want arbitrary-precision numbers